core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -53,6 +59,23 @@
     private Function<? super Throwable, ?> responseCauseSanitizer = DEFAULT_CAUSE_SANITIZER;
     private Function<? super HttpHeaders, ?> responseTrailersSanitizer = DEFAULT_HEADERS_SANITIZER;
 
+    /**
+     * Sets the {@link Logger} to use when logging.
+     * If unset, will use default logger that each logging implementation has.
```

Comment:
It's not so clear what implementation is referring to here. I think we can just simplify it to something like `If unset, a default {@link Logger} will be used.

Reply:
fixed